motr/idx_dix.c

```diff
@@ -1095,6 +1131,11 @@ static void dix_put_ast(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 		flags |= COF_OVERWRITE;
 	if (oi->oi_flags & M0_OIF_SYNC_WAIT)
 		flags |= COF_SYNC_WAIT;
+	if (oi->oi_flags & M0_OIF_CROW)
+		flags |= COF_CROW;
+	if (oi->oi_flags & M0_OIF_SKIP_LAYOUT)
+		flags |= COF_SKIP_LAYOUT;
```

Comment:
Use helper function for setting the flags.

Reply:
Done